sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -740,9 +750,12 @@ public Key restoreKey(byte[] backup) {
      * <p>Lists the deleted keys in the key vault and for each deleted key prints out its recovery id.</p>
      * {@codesnippet com.azure.keyvault.keys.keyclient.listDeletedKeys}
      *
-     * @return A {@link List} containing all of the {@link DeletedKey deleted keys} in the vault.
+     * <p>Iterate over the lists the deleted keys by each page in the key vault and for each deleted key prints out its recovery id.</p>
```

Comment:
Description is fine, But the header should stay too.
`<p><strong>Code Samples to iterate Deleted Keys by page</strong></p>`